jaxrs-tck/pom.xml

```diff
@@ -35,8 +35,8 @@
     </parent>
 
     <properties>
-        <maven.compiler.source>11</maven.compiler.source>
-        <maven.compiler.target>11</maven.compiler.target>
+        <maven.compiler.source>21</maven.compiler.source>
```

Comment:
Is the `maven.compiler.source` still needed if you set the `maven.compiler.release` ?

-- Nicolas

Reply:
You're correct. Some old habits die hard, I'll remove that :)